src/pages/over-signalen.md

```diff
@@ -5,7 +5,7 @@ template: "default"
 
 ## Samenwerken aan Signalen
```

Comment:
The page /over-signalen is not available from the website. I think we should remove this file from the repository.

Reply:
i'll leave the file in this PR we can delete it seperately